packages/migrations/src/Component/Doctrine/Migrations/Configuration.php

```diff
@@ -129,13 +133,13 @@ public function getMigrationsToExecute($direction, $to)
 
     /**
      * @param \Doctrine\DBAL\Migrations\Version $version
-     * @param \Doctrine\DBAL\Migrations\Version[] $migratedVersions
+     * @param string[] $migratedVersions
      * @return bool
      */
     private function shouldExecuteMigration(Version $version, array $migratedVersions)
     {
         foreach ($migratedVersions as $migratedVersion) {
-            if ($version->getVersion() === $migratedVersion->getVersion()) {
+            if ($version->getVersion() === $migratedVersion) {
```

Comment:
I guess we could simplify the whole method using `!in_array($version->getVersion(), $migratedVersions, true)`, just like it was before #1040...